### PR TITLE
Empower Event list to call ListEvent

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR gets rid of dummy events data and changes it to fetch the actual data.

**Which issue(s) this PR fixes**:

Ref https://github.com/pipe-cd/pipecd/issues/2611

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/src/modules/events/index.ts [hunk 1]
```diff
@@ -181,6 +141,7 @@ export const eventsSlice = createSlice({
         const events = action.payload.events;
         if (events.length < FETCH_MORE_ITEMS_PER_PAGE) {
           state.hasMore = false;
+          // TODO: Enable to fetch more Events even if the last response is none
```
Comment (khanhtc1202) [inline pkg/app/web/src/modules/events/index.ts:144]: Should be `disable`?

Comment (khanhtc1202) [reply pkg/app/web/src/modules/events/index.ts:144]: Ah I was wrong, please ignore this 🙏

File: pkg/app/web/src/api/events.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const opts = new ListEventsRequest.Options();
+    opts.setStatusesList(options.statusesList);
+    opts.setName(options.name);
+    req.setOptions(opts);
+    req.setPageSize(pageSize);
+    req.setCursor(cursor);
+    req.setPageMinUpdatedAt(pageMinUpdatedAt);
+    options.labelsMap.forEach((label) => {
+      opts.getLabelsMap().set(label[0], label[1]);
+    });
```
[the remark this answers is not in the document]
Comment (nakabonne) [reply pkg/app/web/src/api/events.ts]: 👍

[the remark this answers is not in the document]
Comment (nakabonne) [reply pkg/app/web/src/api/events.ts]: Applied

**Comment by nakabonne** [comment]
Revised, PTAL ;)

**Comment by khanhtc1202** [comment]
Nice
/approve

**Pull request merged**